packages/data-table-server/src/dataTableService/DataTableServiceBuilder.ts

```diff
@@ -51,7 +52,9 @@ const internalDataTableServiceBuilders = {
   entity_relations: () => new DataTableServiceBuilderForType(EntityRelationsDataTableService),
 };
 
-const userDefinedDataTableServiceBuilders = {};
+const userDefinedDataTableServiceBuilders = {
```

Comment:
Since these all have the same builders, why the differentiation between internal and user defined? Logically they are the same right? It just so happens that some of them target tupaia data and some of them target other data.

Could we achieve the same effect but putting them all in one services folder and then maybe prefixing the tupaia ones with TupaiaX?